### PR TITLE
map height

### PR DESCRIPTION
**Pull request by chloeleichen**
@RacingTadpole , the `var cContainer = document.getElementById('cesiumContainer');` bit probably could be done better? let me know if you need me to change anything

### CODE REVIEW

File: lib/ViewModels/ChartPanelViewModel.js [hunk 1]
```diff
@@ -12,6 +12,9 @@ var combineData = require('../Core/combineData');
 var loadView = require('../Core/loadView');
 var removeView = require('../Core/removeView');
 var ChartViewModel = require('./ChartViewModel');
+var cContainer = document.getElementById('cesiumContainer');
```
Comment (RacingTadpole) [inline lib/ViewModels/ChartPanelViewModel.js]: Hi, thanks for this Chloe - looks good. Yes, is there a way to get the cesium container direct from the terria object?

File: lib/ViewModels/ChartPanelViewModel.js [hunk 2]
```diff
@@ -90,7 +102,8 @@ var ChartPanelViewModel = function(options) {
             return [
                 new ChartViewModel(this, this, combineData(dataPairArrays), {
                     showRangeSelector: true,
-                    canDownload: true
+                    canDownload: true,
+                    height: this.height - 60
```
Comment (RacingTadpole) [inline lib/ViewModels/ChartPanelViewModel.js]: Also is there a way to not hardcode this 60? Eg. if it's all due to the chart panel title bar, let's define that height somewhere and use that.

File: lib/ViewModels/ChartPanelViewModel.js (hunk 1 above)
Comment (chloeleichen) [inline lib/ViewModels/ChartPanelViewModel.js]: I think so, but i dunno how, @kring must know :dart:

Comment (kring) [inline lib/ViewModels/ChartPanelViewModel.js]: `Terria` knows the `currentViewer`, which surely has a reference to its DOM element somehow, but it's not really exposed in a useful way.  We could probably expose it.

**Comment by RacingTadpole** [comment]
@chloeleichen - can I put this back on your radar? Just a few little things above. Thanks!

**Comment by chloeleichen** [comment]
@RacingTadpole done :smile:

File: lib/ViewModels/ChartPanelViewModel.js [hunk 3]
```diff
@@ -49,6 +57,16 @@ var ChartPanelViewModel = function(options) {
 
     updateMaxDimensions();
 
+    knockout.getObservable(this, 'isVisible').subscribe(function(isVisible) {
+        var cesiumContainer = this.terria.currentViewer.viewer.container;
```
Comment (RacingTadpole) [inline lib/ViewModels/ChartPanelViewModel.js:61]: Thanks @chloeleichen! Unfortunately this doesn't work with Leaflet... (it doesn't have a `viewer` property) - any ideas?

Comment (chloeleichen) [inline lib/ViewModels/ChartPanelViewModel.js:61]: `document.getElementById()?`

Comment (RacingTadpole) [inline lib/ViewModels/ChartPanelViewModel.js:61]: Ha! That's what you removed in the last commit! I've just poked around in terriajs's Leaflet class and found this:
`map.getContainer()`, ie. we can use `this.terria.currentViewer.map.getContainer()` here - this returns the DOM element we need.

Comment (chloeleichen) [inline lib/ViewModels/ChartPanelViewModel.js:61]: sounds cool, do you want me to do that or you've already done it?

Comment (RacingTadpole) [inline lib/ViewModels/ChartPanelViewModel.js:61]: I can do it... actually I'm thinking that we should really define a `getContainer` method on `currentViewer` directly, and then use that.

**Pull request merged**